app/cli/internal/policydevel/lint.go

```diff
@@ -113,31 +121,32 @@ func Lookup(absPath, config string, format bool) (*PolicyToLint, error) {
 	return policy, nil
 }
 
-// Performs a one-level directory lookup to find .yaml/.yml or .rego files.
-func scanDirectory(policy *PolicyToLint, dirPath string) error {
-	files, err := os.ReadDir(dirPath)
-	if err != nil {
-		return fmt.Errorf("reading directory: %w", err)
-	}
-
-	var foundValidFile bool
-	for _, file := range files {
-		if file.IsDir() {
+// Loads referenced rego files from all YAML files in the policy
+func loadReferencedRegoFiles(policy *PolicyToLint) error {
```

Comment:
very nit: the argument could be a receiver instead `func (policy *PolicyToLint) loadReferenceRegoFiles() {`